src/pages/Home.vue

```diff
@@ -53,6 +53,10 @@ export default {
     async activeWorkspace() {
       if (this.isUserApproved) await this.fetchAllPlioIds();
     },
+    isUserApproved(value) {
+      // fetch plios again if user approval status changes
+      if (value) this.fetchAllPlioIds();
+    },
```

Comment:
why has this been added?

Reply:
Eg- I'm on "waitlist". I log in -> I'm shown the waitlist waala warning message
API se the superuser approves me -> frontend mein `isUserApproved` becomes `true`. BUT `hasAnyPlios` is still false and the data hasn't been fetched because we don't fetch data when the user is unapproved. 
So as soon as the approval status changes, we need to fetch again.

And calling `this.fetchAllPlioIds` won't be too expensive  if not doing it for the first time. As we store all plios ka data in the vuex store, we can reuse that if it exists on the store.